### PR TITLE
Migrate from printing error logs to using the tokio tracing crate for logging.

### PR DESCRIPTION
**Pull request by brandonpollack23**

### CODE REVIEW

File: src/cmd/dut.rs [hunk 1]
```diff
@@ -142,16 +146,16 @@ fn run_dut_vnc(args: &ArgsVnc) -> Result<()> {
...
     let mut child_novnc = target.start_port_forwarding(web_port, 6080, "novnc")?;
 
-    eprintln!("To use VNC via web browser, please open:");
-    eprintln!("  http://localhost:{web_port}/vnc.html");
-    eprintln!("To connect via VNC client directly, use:");
-    eprintln!("  xtightvncviewer -encodings raw localhost:{vnc_port}");
+    warn!("To use VNC via web browser, please open:");
+    warn!("  http://localhost:{web_port}/vnc.html");
+    warn!("To connect via VNC client directly, use:");
+    warn!("  xtightvncviewer -encodings raw localhost:{vnc_port}");
```
Comment (mhiramat) [inline src/cmd/dut.rs:158]: This is not a warning message but an information for user to tell how to open VNC.

**Comment by brandonpollack23** [comment]
agreed but I anticipate some users/systems (maybe some time ahead from now)
might filter out "info" and this is important for the user to see so I made
it "warn" level to preserve the message.  As in:

Warning: Just so you know how to do this do that

rather than

Warning: Something is wrong!

**Comment by brandonpollack23** [comment]
lil' ping

File: src/cmd/dut.rs (hunk 1 above)
Comment (hikalium) [reply src/cmd/dut.rs:158]: I think it can be warn to ensure the message is printed out to the users.

**Review by hikalium — APPROVED** [review]

**Pull request merged**